en/test_and_ci/continous_integration.md

```diff
@@ -1,3 +1,77 @@
-# PX4 Continuous Integration
+# PX4 Continuous Integration / Automation
 
-PX4 builds and testing are performed using GitHub actions and a Jenkins server.
\ No newline at end of file
+PX4 has multiple automated CI build that runs to keep different systems in sync. This document describes how they are configured & connected, to hopefully clear-up confusion in how everything is connected!
+
+:::note
+CI scripts are subject to change, and the links below may get broken in the future. If that happens, please contribute the fix for the documentation!
+:::
+
+## PX4-Autopilot Pull Request CI tests
+For every [Pull-Request](https://github.com/PX4/PX4-Autopilot/pulls), there are automated CI that checks the PR.
+Most of the workflows are done with Github workflows, which can be found [here](https://github.com/PX4/PX4-Autopilot/tree/main/.github/workflows).
+
+- `compile_nuttx.yml`: Builds NuttX flight controller board targets. It can detect Flash overflows and code syntax errors
+- `sitl_tests.yml`: Executes [MAVSDK SITL Tests](./integration_testing_mavsdk.md). It can detect functional bugs in the code (e.g. VTOL doesn't return to home)
+- `mavros_mission_tests.yml`: Executes Mission plans using MAVROS. It can check for functional compatibility between MAVROS and PX4
+- `checks.yml`: Runs various make targets (using `make` command). The commands for each targets are documented in the [Makefile](https://github.com/PX4/PX4-Autopilot/blob/main/Makefile)
+  - `check_format`: Checks the coding style format using Clang Tidy ?
```

Comment:
@dagar Is this different from the `clang-tidy.yml` part below?

Reply:
@MaEtUgR This section might be affected by linter changes

Reply:
@hamishwillee Certainly, thanks for the heads-up. We won't change it over night. Style is currently exclusively checked by `astyle`, see https://github.com/PX4/PX4-Autopilot/blob/3b2d76657312d47e6495736dfd4d694b546497d5/Tools/astyle/fix_code_style.sh#L8-L11

Reply:
OK, so this is good for now. Thanks!